samples/event_manager_profiler/src/modules/module_b.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+
+#include "five_sec_event.h"
+#include "one_sec_event.h"
+
+#define MODULE module_b
+
+
+static bool event_handler(const struct event_header *eh)
+{
+	if (is_one_sec_event(eh)) {
+		struct one_sec_event *ose = cast_one_sec_event(eh);
+
+		if (ose->five_sec_timer == 5) {
```

Comment:
Consider defining the constant: `#define TIMER_THRESHOLD 5`

Reply:
Thought about it but realized that we have five_sec event so then we shouldn't change it and that's why I have it hardcoded.